src/arrayMethodSort.js

```diff
@@ -5,7 +5,42 @@
  */
 function applyCustomSort() {
   [].__proto__.sort2 = function(compareFunction) {
-    // write code here
+    let compare = compareFunction;
+    let count;
+
+    if (!compare) {
+      compare = function(a, b) {
+        const strA = a.toString();
+        const strB = b.toString();
+
+        if (strA > strB) {
+          return 1;
+        }
+
+        if (strA < strB) {
+          return -1;
+        }
+
+        return 0;
```

Comment:
```suggestion
        return strA > strB;
```
You can simplify your code. If `strA > strB`, the function will return `true`. Otherwise - `false`. 
`true` is always `greater than` 0, so it is very useful in your comparison `f (compare(prev, cur) > 0)`